cuda/apps/src/kinect_debayering.cpp

```diff
@@ -67,18 +67,17 @@ class SimpleKinectTool
     	}
     	imshow ("test", cv_image);
     }
-    
+
     void run (const std::string& device_id)
     {
 	    cv::namedWindow("test", CV_WINDOW_AUTOSIZE);
       pcl::Grabber* interface = new pcl::OpenNIGrabber(device_id);
 
       std::function<void (const openni_wrapper::Image::Ptr& image)> f = std::bind (&SimpleKinectTool::cloud_cb_, this, _1);
 
-      boost::signals2::connection c = interface->registerCallback (f);
```

Comment:
The `registerCallback` clearly has side effects. You can't just remove it. Please take note for other files too